modules/va_facilities/app/controllers/va_facilities/docs/metadata_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module VaFacilities
```

Comment:
curious why facilities is part of this PR .. are you just adding metatdata for anything that currently has multiple versions?

Reply:
Jeff had reached out to me to let me know that they had a second version of their API and since I was already working versions I told him I'd include it. The idea in our implementation though is to have the developer portal support both a single swagger url or a metadata end point with multiple which will make it easier to on board other projects etc

Reply:
Thanks for including this. It certainly wasn't our intention for you to take on any extra work, but we appreciate it!